diff --git a/node_modules/@react-navigation/drawer/lib/module/views/modern/Drawer.js b/node_modules/@react-navigation/drawer/lib/module/views/modern/Drawer.js
index 3aa1baa..a5adcfd 100644
--- a/node_modules/@react-navigation/drawer/lib/module/views/modern/Drawer.js
+++ b/node_modules/@react-navigation/drawer/lib/module/views/modern/Drawer.js
@@ -142,7 +142,7 @@ export default function Drawer(_ref) {
     open,
     isUserInitiated: false
   }), [open, toggleDrawer]);
-  const onGestureEvent = useAnimatedGestureHandler({
+  const onGestureEvent = ((() => () => {}))({
     onStart: (event, ctx) => {
       ctx.hasCalledOnStart = false;
       ctx.startX = translationX.value;
